src/v/cloud_io/remote.h

```diff
@@ -126,6 +128,21 @@ class remote_api {
       = 0;
 };
 
+enum class api_activity_type {
+    object_upload,
```

Comment:
`post_object_upload_attempt`? I see that we notify in case of failure too. And it would be useful to know whether the notification is pre/post.

Reply:
I changed the code so all notifications are post and errors are not notified.